core/arch/x86/decode_table.c

```diff
@@ -6691,6 +6689,14 @@ const instr_info_t evex_W_extensions[][2] = {
     {OP_vmovdqu32, 0xf30f7f10,"vmovdqu32",Wex,xx,KEw,Vex,xx,mrm|evex,x,END_LIST},
     {OP_vmovdqu64, 0xf30f7f50,"vmovdqu64",Wex,xx,KEw,Vex,xx,mrm|evex,x,END_LIST},
   },
+  {    /* evex_W_ext 14 */
+    {OP_vmovlps, 0x0f1210, "vmovlps", Vq_dq, xx, Hq_dq, Wq_dq, xx, mrm|evex|reqL0, x, tevexw[15][0]}, /*"vmovhlps" if reg-reg */
+    {INVALID, 0x0f1210,"(bad)", xx,xx,xx,xx,xx,no,x,NA},
```

Comment:
Not required to work, but to be consistent, this needs to be 0x0f1250. I will update before merging.

Reply:
Done.